fvm/transactionInvoker.go

```diff
@@ -220,14 +220,6 @@ func (i *TransactionInvoker) Process(
 
 			return feesError
 		}
-	} else {
-		// transaction is ok, log as successful
-		i.logger.Info().
-			Str("txHash", txIDStr).
-			Uint64("blockHeight", blockHeight).
-			Uint64("ledgerInteractionUsed", sth.State().InteractionUsed()).
-			Int("retried", proc.Retried).
```

Comment:
I saw this PR late, unfortunately removal of ledgerInteractionUsed broke some metrics we were interested in, anyway we can put them back @zhangchiqing